package.json

```diff
@@ -20,23 +20,25 @@
     "jasmine-core": "^3.5.0"
   },
   "devDependencies": {
-    "conventional-changelog": "1.1.7",
-    "conventional-changelog-core": "1.9.3",
-    "eslint-config-standard": "^5.1.0",
-    "eslint-plugin-promise": "^1.1.0",
-    "eslint-plugin-standard": "^1.3.2",
+    "conventional-changelog": "^3.1.18",
+    "conventional-changelog-core": "^4.1.4",
+    "eslint-config-standard": "^14.1.0",
+    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
```

Comment:
We could probably just use standard directly or https://www.npmjs.com/package/grunt-standard, but I'm not sure if/when I'll have the time. Just thinking out loud

Reply:
yeah, I'd even like to have a karma-standard based on grunt-standard and leverage into all the karma-runner projects uniformly.